cmd/soroban-cli/src/commands/contract/deploy/wasm.rs

```diff
@@ -218,25 +228,43 @@ impl NetworkRunnable for Cmd {
             salt,
             &key,
         )?;
+
         if self.fee.build_only {
+            output.check("Done!");
             return Ok(TxnResult::Txn(txn));
         }
 
+        output.info("Simulating deploy transaction…");
+
         let txn = client.simulate_and_assemble_transaction(&txn).await?;
         let txn = self.fee.apply_to_assembled_txn(txn).transaction().clone();
+
         if self.fee.sim_only {
+            output.check("Done!");
             return Ok(TxnResult::Txn(txn));
         }
+
+        output.globe("Submitting deploy transaction…");
+        utils::log_transaction(&output, &txn, &network, true)?;
```

Comment:
Is the quiet supposed to be the last argument? If so, wouldn't the `Output` print method silence it anyway?

Reply:
No, the last argument is whether we want to display the explorer link or not. When only simulating, we won't be submitting transaction, so we won't have a link.

That said, I forgot to send whether is only building/simulating, rather than passing `true`.

Reply:
oh, in this case I'm passing `true` because `build_only` and `sim_only` early return, so there's no need to pass those values.